### ISSUE TITLE
Bug in fine alignment warning

### ISSUE THREAD

**Issue by lanery** [delmic/odemis#1549]
In the warning message for when the fine alignment encounters an unusually large rotation, scale, etc. it gives the following

```python
else:  # Generic warning
    logging.warning(u"The fine alignment values are very large, try on a different place on the sample. "
                               u"mag correction: %f, rotation: %f°, shear: %f, X/Y scale: %f",
                               opt_scale, rot_deg, shear, scaling_xy)
```
https://github.com/delmic/odemis/blob/master/src/odemis/gui/cont/acquisition.py#L1068

which is fine except that `scaling_xy` is a tuple I think (or at least not a float).

**Comment by pieleric** [comment]
Indeed. Thanks for the report, and such precise diagnostic! 
PR #1550 should fix it.

**Comment by pieleric** [comment]
Fixed in v3.1.0 (and v3.0.7).
